src/thread/thread_entry.cpp

```diff
@@ -77,7 +77,7 @@ namespace cubthread
   entry::entry ()
   // public:
     : index (-1)
-    , type (TT_WORKER)
+    , type (TT_MASTER)
```

Comment:
@ribram Somehow I feel this change is either incomplete or incorrect.
May be completed/corrected with:
- change `TT_MASTER` to `TT_MAIN`, in order to not confuse with `cub_master` related functionality?
- add an explicit `thread_type` to the `cubthread::entry` ctor and explicitly change all over - or request everyone to explicitly set a thread type
With this change, from now on, all non-initialized thread entries in thread manager will be `TT_MASTER` whereas they were `TT_WORKER` (more proper imho).

Reply:
> change TT_MASTER to TT_MAIN, in order to not confuse with cub_master related functionality?

It does not matter, the difference is in the context. This is the master thread of cub_server process, the cub_master is entirely another process.

>add an explicit thread_type to the cubthread::entry ctor and explicitly change all over - or request everyone to explicitly set a thread type
With this change, from now on, all non-initialized thread entries in thread manager will be TT_MASTER whereas they were TT_WORKER (more proper imho).

What are your thoughts on initializing as TT_NONE in the cubthread::entry constructor and setting TT_MASTER type in `cubthread::initialize()`?